src/memote/consistency.jl

```diff
@@ -296,6 +296,7 @@ function model_is_consistent(model, optimizer; config = memote_config)
     termination_status(opt_model) == OPTIMAL
 end
 
+<< << << < HEAD:src/memote/consistency.jl
```

Comment:
what is up with this?

Reply:
Ah sorry, git automatically wanted to add the gene annotation tests from the previous version of this PR to the consistency.jl file when I rebased for some reason. I moved them to the correct file but accidentally missed that line, I'll remove it now.